nautobot_ssot_aci/diffsync/adapters/aci.py

```diff
@@ -264,25 +265,18 @@ def load_interfacetemplates(self):
 
     def load_interfaces(self):
         """Load interfaces from ACI."""
+        devicetype_file_path = os.path.join(os.path.dirname(__file__), "..", "device-types")
         interfaces = self.conn.get_interfaces(
             nodes=self.devices,
         )
 
         for node in self.devices:
             # Load management and controller interfaces from YAML files
 
-            fn = os.path.join(
-                os.getcwd(), "nautobot_ssot_aci", "diffsync", "device-types", f"{self.devices[node]['model']}.yaml"
-            )
+            fn = os.path.join(devicetype_file_path, f"{self.devices[node]['model']}.yaml")
             if os.path.exists(fn):
                 device_specs = load_yamlfile(
-                    os.path.join(
-                        os.getcwd(),
-                        "nautobot_ssot_aci",
-                        "diffsync",
-                        "device-types",
-                        f"{self.devices[node]['model']}.yaml",
-                    )
+                    os.path.join(devicetype_file_path, f"{self.devices[node]['model']}.yaml",)
```

Comment:
why not reusing `fn`?
```suggestion
                    fn
```

Reply:
Good call. Fixed.